### DISCUSSION TITLE
Settings view

### DISCUSSION THREAD

**Discussion by shuvashish76** [TeamAmaze/AmazeFileManager#3174] [Ideas]
#3140 merged & ready for 3.7 release.
Tested debug version 3.6.7 (107) from Amaze's [Telegram channel](https://t.me/TeamAmaze)

![image](https://user-images.githubusercontent.com/48226312/154650002-303b796d-e83d-4e28-8772-c30191ef4d0c.png)

Grouped settings looks really nice but I prefer the old view with all preferences in a single page. I'm sure some ppl would like to see everything in 1 page which gives a pleasant secure feeling.
Some ppl on playstore review requested for import/export Amaze settings option (cause too many options).

![Screenshot_2022-02-18-14-54-59](https://user-images.githubusercontent.com/48226312/154655024-5ac57f93-a456-4173-8537-672837620dc6.jpeg)

source: Amaze FM's playstore review

Now it creates more confusion (& insecure feeling) with Grouped view.

---

Suggestion :
Add an option to view Settings preference as Grouped or Flat view.

**Comment by EmmanuelMess** [comment]
Against, we would have to maintain and test two views, and the flat view would be too long.

**Comment by shuvashish76** [comment]
> Some ppl on playstore review requested for import/export Amaze settings option (cause too many options).

#3176

> Against, we would have to maintain and test two views, and the flat view would be too long.

Agreed 👍🏿
Suggestion: Searchable preferences (#3178)
